open_instruct/ground_truth_utils.py

```diff
@@ -139,7 +139,7 @@ def verify_flan_sample(model_output, ground_truth_answer):
 
 def soft_format_reward_func(responses: list[str]) -> list[float]:
```

Comment:
Would agree with nathan that it feels like reward weight should be a param (and maybe even what pattern you are looking for?), to help with tuning reward stuff in the future.